app/build.gradle

```diff
@@ -71,6 +73,10 @@ dependencies {
     }
     implementation 'com.evernote:android-job:1.2.5'
     implementation 'com.android.support.constraint:constraint-layout:1.1.0'
+    // EmojiCompat
+    implementation "com.android.support:support-emoji:$supportLibraryVersion"
+    implementation "com.android.support:support-emoji-bundled:$supportLibraryVersion"
```

Comment:
NOPE this bundles the noto font into the apk we do not want this

Reply:
Oh. I thought I deleted that line 😂